sections/transformers.tex

```diff
@@ -0,0 +1,55 @@
+\subsection{Which problems transformers solve}
+The point of transformers was to overcome the problems that previous state of the art architectures have faced. Transformers aim to get the best features from the RNN and Convolutional Neural Network (CNN) models.
+
+The RNN model has two notable weaknesses. First, its inability to learn long term patterns, due to the exploding/vanishing gradient problems that occur during backpropagation.
+Second, its recurrent connection is also a weakness. This is because, as information is propagated along a sequence, you cannot compute the cell at time step $i$ until you've computed the cell at time step $i-1$.
```

Comment:
```suggestion
Secondly, its recurrent connection is also a weakness. This is because it is not possible to compute the cell at time step $i$ until the cell at time step $i-1$ has been computed as information is propagated along a sequence.
```